### ISSUE TITLE
Questions relating to dataset splits?

### ISSUE THREAD

**Issue by liyuxuan89** [SHI-Labs/SGL-Retinal-Vessel-Segmentation#1]
Thanks for open sourcing the codes. 
I have a question relating to dataset splits.
The training config data_range is set to '1-20/1-8'. Does that mean the training set overlap with the test set?

**Comment by yzhouas** [comment]
Hi Yuxuan,

We separate the fixed training and testing set, so the number range indicates the samples in different sets. For example, 1-20 means the training set, while 1-8 means the samples in the testing set. Therefore, they are not overlapped.
